locales/tr.js

```diff
@@ -1,36 +1,36 @@
-  "Server ID": "Sunucu Kimliği",
-  "Cluster ID": "Cluster ID",
-  "Shard ID": "Shard Kimliği",
-  "Default channel": "Normal kanal",
+  "Server ID": "Sunucu ID'si",
+  "Cluster ID": "Cluster ID'si",
+  "Shard ID": "Shard ID'si",
+  "Default channel": "Varsayılan kanal",
   "Invite Link": "Davet Linki",
   "Main Language": "Ana Dil",
-  "I'm handling your command": "Komudunu ele alıyorum.",
-  "This server is set to private. \nSet to public from the dashboard (https://disboard.org/dashboard) to bump this server": "Bu sunucu **özel** olarak ayarlanmıştir. \nKontrol panelinden (https://disboard.org/dashboard) sunucu patlatmak için herkese açık hale getirin.",
-  "This server has not been added to DISBOARD yet. \nAdd this server from the dashboard (https://disboard.org/dashboard)": "Bu sunucu DISBOARD'a henüz eklenmedi. Kontrol panelinden bu sunucuyu ekle.",
-  "This server is not listed because the invite has been deleted": "This server is not listed because the invite has been deleted",
-  "Create invite with `%s` command": "Create invite with `%s` command",
-  "Command failed": "Command failed",
-  "DISBOARD API server seems to be down": "DISBOARD API server seems to be down",
-  "You are rate limited. Please wait %s seconds": "You are rate limited. Please wait %s seconds",
-  "**How do I add my server to DISBOARD?**\n\n 1. Login on DISBOARD website\n 2. Go to Dashboard\n 3. Click \"Add new server\"\nFill out your server info and save it. You will be redirected to Discord's authorization screen. If not, click the \"Add Bot\" button on the server edit page.": "**How do I add my server to DISBOARD?**\n\n 1. Login on DISBOARD website\n 2. Go to Dashboard\n 3. Click \"Add new server\"\nFill out your server info and save it. You will be redirected to Discord's authorization screen. If not, click the \"Add Bot\" button on the server edit page.",
-  "Support server: <%s>": "Support server: <%s>"
+  "I'm handling your command": "Komudun ile ilgileniyorum",
+  "This server is set to private. \nSet to public from the dashboard (https://disboard.org/dashboard) to bump this server": "Bu sunucu **özel** olarak ayarlanmıştır. \nKontrol panelinden (https://disboard.org/dashboard) sunucuyu öne çıkarmak için herkese açık hale getirin.",
+  "This server has not been added to DISBOARD yet. \nAdd this server from the dashboard (https://disboard.org/dashboard)": "Bu sunucu DISBOARD'a henüz eklenmedi. \nKontrol panelinden (https://disboard.org/dashboard) bu sunucuyu ekle.",
+  "This server is not listed because the invite has been deleted": "Bu sunucu listelenmedi çünkü davet silindi",
+  "Create invite with `%s` command": "`%s` komudu ile davet oluşturun",
+  "Command failed": "Komut kullanılamadı",
+  "DISBOARD API server seems to be down": "DISBOARD API sunucusu çöktü gibi görünüyor",
+  "You are rate limited. Please wait %s seconds": "Hız sınırını aştınız. Lütfen %s saniye bekleyin"
```

Comment:
Same thing about the informal language here, "**Hız sınırını aştın. Lütfen %s saniye bekle.**"